package.json

```diff
@@ -23,9 +23,9 @@
     "testing/"
   ],
   "scripts": {
-    "build": "npm run tsc.scripts && npm run tsc.prod && npm run rollup.prod.ci",
-    "build.esbuild": "npm run clean && npm run tsc.scripts && npm run tsc.prod && node scripts/build/esbuild/build.js --prod --ci",
-    "build.esbuild.watch": "npm run build.esbuild -- --watch",
+    "build.rollup": "npm run tsc.scripts && npm run tsc.prod && npm run rollup.prod.ci",
```

Comment:
Do we need `npm run clean` at the start of this?

Reply:
I don't think so, since the Rollup build clears out the directories here:

https://github.com/ionic-team/stencil/blob/1eec9a82a4d623d46228e84798adbf2f88d7f3e8/scripts/build.ts#L44-L52

Reply:
it does not - went ahead and just added it because why not

although I'll point out this is our current build script, so we're currently not clearing out `scripts/` stuff between builds

although 🤔 maybe that's intentional so that the caching business that happens in there sticks around?

Reply:
Yeah, exactly - I typically run it (`clean`) whenever I switch branches locally (`npm run clean && npm ci && npm run build`), but it does slow down some overall build time

Reply:
yeah, ok I'm going to go ahead and leave the command as it is on `main` for now